README.md

```diff
@@ -1,12 +1,12 @@
-![Travis CI badge](https://api.travis-ci.com/rust-lang/rustc-guide.svg?branch=master)
+![Travis CI badge](https://api.travis-ci.com/rust-lang/rustc-dev-guide.svg?branch=master)
```

Comment:
Will Travis CI automatically change this on their part?

Reply:
The GITHUB_TOKEN may need to be updated on Travis after the move?  

https://github.com/rust-lang/rustc-guide/blob/master/.travis.yml#L26

Reply:
Hmm... I don't know actually... we may need to reset up the ci...

Reply:
I added a checklist item for this.